docs/gettsim_objects/input_variables.rst

```diff
@@ -101,3 +101,29 @@ household.
 +-------------------------+---------------------------------------------+--------------+
 | _`priv_rente_m`         | Amount of monthly private pension           | float        |
 +-------------------------+---------------------------------------------+--------------+
+| _`weiblich`             | If gender is female                         | bool         |
++-------------------------+---------------------------------------------+--------------+
+| _`rententrechtl_zeit`   | Sum of all times relevant for retirement    | float        |
++-------------------------+---------------------------------------------+--------------+
+| _`pflichtbeitragszeit`  | Months of mandtory contributions            | float        |
++-------------------------+---------------------------------------------+--------------+
+| _`freiw_beitragszeit`   | Months of voluntary contributions           | float        |
++-------------------------+---------------------------------------------+--------------+
+|| _`anrechnungszeit`     || Months of sickness, pregnancy/Mutterschutz,|| float       |
```

Comment:
Are the vertical bars supposed to be doubled up?

Reply:
I did this when there where multiple lines of description (copied this from earlier entries). Not sure if that is necessary.

Reply:
Ah, excellent, looks great! https://gettsim--396.org.readthedocs.build/en/396/gettsim_objects/input_variables.html

And I just learned something! Thanks!